src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -834,6 +834,8 @@ s32 EnKo_ChildStart(EnKo* this, PlayState* play) {
         case ENKO_TYPE_CHILD_FADO:
             return func_80A97E18(this, play);
     }
+    //! @bug No return value. This function does not set v0 so the value returned
+    //! will be the return value of EnKo_GetForestQuestState.
```

Comment:
I'm skeptical that the things in this file are bugs. Is there a way to spawn an actor with an invalid type in-game (without exploiting some other bug, like SRM)? If not then undefined behavior cannot occur, right? I'm not sure that assuming that the actor has a valid type is so different than assuming that some parameter is not null, for example.

Reply:
what about adding some kind of UNREACHABLE annotation, would that satisfy GCC?

Reply:
I've changed these to use `__builtin_unreachable()` with a comment explaining the assumptions the original code makes

Reply:
thanks, that does seem better to me now that it's in the code. Should we factor about the __GNUC__ check to something like `UNREACHABLE();`, or is that too much?

Reply:
it might solve the indentation weirdness too, heh

Reply:
Done, it did seem to solve the indentation too 😌